textBasedAdventure.js

```diff
@@ -1,5 +1,25 @@
 const readline = require('readline-sync')
 
-let nameInput = readline.question("Enter your name: ")
+const  play = () => {
+    let userInput;
+    userInput = readline.question('Would you like to play?') 
+    while (userInput !== "yes") {
+        console.log("Really?")
+        userInput = readline.question("Would you like to play? Enter yes or no?")
+    } 
+    if (userInput === "yes"); {
+            console.log("You've made the right choice.")
+    userInput = readline.question("Are you having a good day or a bad day?")
+    while (userInput === "good day"||"Good Day"||"Good"||"good") {
```

Comment:
This is not valid JS code
You need:
while (userInput === 'good day' || userInput === 'Good Day' || userInput === 'Good' || userInput === 'good')

Instead of checking for each possibility with upper case or lower case you could use the method `toLowerCase()`